README.md

```diff
@@ -1367,7 +1368,55 @@ export default () => (
 );
 ```
 
-[Google Docs for Social Profile](https://developers.google.com/search/docs/data-types/social-profile)
```

Comment:
Could add this back in and move it to be at the end of [this section](https://github.com/garmeeh/next-seo#social-profile) just before the News heading.